src/Sdk/WebApi/WebApi/Contracts.cs

```diff
@@ -126,6 +127,46 @@ public class CreateMetadataResponse
         public bool Ok;
     }
 
+    [DataContract]
+    [JsonObject(NamingStrategyType = typeof(SnakeCaseNamingStrategy))]
+    public class StepsUpdateRequest
+    {
+        [DataMember]
+        public IEnumerable<Step> Steps;
+        [DataMember]
+        public long ChangeOrder;
+        [DataMember]
+        public string WorkflowJobRunBackendId;
+        [DataMember]
+        public string WorkflowRunBackendId;
+    }
+
+    [DataContract]
+    [JsonObject(NamingStrategyType = typeof(SnakeCaseNamingStrategy))]
+    public class Step
+    {
+        [DataMember]
+        public string ExternalId;
+        [DataMember]
+        public int Number;
+        [DataMember]
+        public string Name;
+        [DataMember]
+        public Status Status;
+        [DataMember]
+        public string StartedAt;
+        [DataMember]
+        public string CompletedAt;
+    }
+
+    public enum Status
```

Comment:
It may be helpful to link to somewhere to indicate why were are choosing these particular status numbers? Otherwise the context on the strange ordering of the numbers may get lost. Unless there is already an explanation in results.

Reply:
We have a list of `Status` and the numbers are consecutive, but only those are relevant to steps.  🤷🏼